CONTRIBUTING.md

```diff
@@ -29,13 +27,10 @@ any guidance or assistance.
 ## Feature Requests
 [feature-requests]: #feature-requests
 
-To request a change to the way that Tremor works, please head over
-to the [RFCs repository](https://github.com/wayfair-tremor/tremor-rfcs) and view the
-[README](https://github.com/wayfair-tremor/tremor-rfcs/blob/master/README.md)
-for instructions.
+To request a new feature, please file it as a new [issue](https://github.com/wayfair-tremor/tremor-vscode/issues/new).
 
 ## Bug Reports
-[bug-reports]: #bug-reports
+[buvscodets]: #bug-reports
```

Comment:
is this a typo?

Reply:
haha yes no idea how that happened. just fixed it!

Reply:
Well spotted!